visualizer/index.html

```diff
@@ -203,7 +242,7 @@
     #bottomSection {
       display: flex;
       flex: 1;
-      flex-direction: column;
+      overflow-x: auto;
```

Comment:
Ok, I just pushed a change that implements what I described above. It's not quite right, because the vertical scrollbar is only visible when `#bottomSection` is scrolled all the way to the right, but I think that's fine for now.

It's a really tricky layout, and we'll have to think about whether there's a better way to solve it.